src/lib/sw.js

```diff
@@ -0,0 +1,48 @@
+importScripts(
+  "https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js",
+);
+
+// TODO(samthor): rollup a list of assets to precache on install.
+// TODO(samthor): match Google Fonts, Firebase and friends.
```

Comment:
I think I have code for that which I stole from the workbox docs:
https://github.com/GoogleChrome/web.dev/blob/workbox/src/lib/service-worker.js#L26